ci/views.py

```diff
@@ -34,6 +34,9 @@
 from django.views.decorators.cache import never_cache
 from ci.client import UpdateRemoteStatus
 import os, re
+from datetime import datetime
+from croniter import croniter
+import pytz
```

Comment:
Is this something we need to add to requirements.txt?

Reply:
croniter and pytz are already in it.  datetime is just the standard library.

Reply:
My mistake, I didn't see pytz (was the only one I had in question)